src/components/atoms/Dropdown/Dropdown.tsx

```diff
@@ -57,32 +56,9 @@ interface DropdownProps {
   noArrow?: boolean;
 }
 
-export const Dropdown: React.FC<DropdownProps> = ({
-  title,
-  className,
-  placement,
-  noArrow,
-  children,
-}) => {
+export const Dropdown: React.FC<DropdownProps> = ({ title, children }) => {
   const value = remap(title);
   const options = React.Children.map(children, remap) ?? ALWAYS_EMPTY_ARRAY;
 
-  const containerClasses = classNames(
```

Comment:
Always nice to see code deleted, but will this cause issues and if not, would it be safe to move to `./src/components/admin` or is it used in attendee side?

Reply:
It is currently used in both places and that's why I'm not moving it. But yes, the classes are safe to remove since there are no styles anywhere.